neo/Network/P2P/Capabilities/FullNodeCapability.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.IO;
+
+namespace Neo.Network.P2P.Capabilities
+{
+    public class FullNodeCapability : NodeCapability
```

Comment:
This full node capability only with start height is quite strange.

Reply:
`FullNodeCapability` means the node has full blockchain data and accept relay.

Reply:
If you are a relay node, why you should provide a 0 or this information?